prow/config/periodics.yaml

```diff
@@ -0,0 +1,51 @@
+periodics:
+- name: push-istio-images-centos-2.2
+  decorate: true
+  cron: "00 03 * * *"  # Run at 3:00 UTC everyday
+  labels:
+    preset-quay-pusher: "true"
+  extra_refs:
+  - org: maistra
+    repo: istio-images-centos
+    base_ref: maistra-2.2
+  spec:
+    containers:
+    - image: "quay.io/maistra-dev/maistra-builder:2.2"
+      imagePullPolicy: Always
+      command:
+      - ./create-images.sh
+      - -h quay.io/maistra
+      - -t 2.2.x-daily-$(date +%Y-%m-%d)
```

Comment:
I don't think this will be expanded. There's no shell here. It's a direct spawn of a process and this will be passed as-is.